pkgs/applications/science/molecular-dynamics/gromacs/default.nix

```
@@ -146,5 +146,6 @@ in stdenv.mkDerivation rec {
     '';
     platforms = platforms.unix;
     maintainers = with maintainers; [ sheepforce markuskowa ];
+    mainProgram = "gmx";
```

```suggestion
    mainProgram = if enableMpi then "gmx_mpi" else "gmx";
```

Good idea, the MPI main executable is `gmx_mpi` though :) But yes, we should set this attribute for Gromacs.